tests/TestCase.php

```diff
@@ -37,6 +37,15 @@ protected function setUp(): void
     protected function getEnvironmentSetUp($app)
     {
         $app['config']->set('firefly.api.enabled', true);
+        $app['config']->set('firefly.user', User::class);
+    }
+
+    /**
+     * Define environment for testing watchers.
+     */
+    protected function usesWatchers($app)
+    {
+        $app->config->set('firefly.features.watchers', true);
```

Comment:
Is there a better way to do this? Maybe do a call to `enableWatchersFeature` inside of the `setUp` methods of the tests that need it? I personally would rather not use `@define-env` in tests.